assets/src/edit-story/components/library/panes/text/karma/textPane.karma.js

```diff
@@ -30,20 +30,26 @@ import { PRESETS } from '../textPresets';
 
 describe('CUJ: Creator can Add and Write Text: Consecutive text presets', () => {
   let fixture;
+  let originalTimeout;
+
   beforeEach(async () => {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 300000;
     fixture = new Fixture();
     await fixture.render();
   });
 
   afterEach(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
```

Comment:
The second test fires off 16 awaits since it is testing staggering text placements. With text sets we are now loading fonts from Google and in every run 10s timeout just isn't enough time for this entire interaction to take place on the CI server.